final release/models.py

```diff
@@ -8,69 +8,67 @@
 from sklearn.svm import LinearSVC
 
 class SklearnModel:
-     """Creates sklearn classifer object.
-     
-     Run train method to train the classifer. Use load method to return
-     the trained model.
-    
+    """Instantize sklearn classifer object.
```

Comment:
"Instantiate"

Reply:
But that's not a description of this class. The description should be more like "Wrapper for sklearn classifiers." and a brief description of what it means to have a `SklearnModel`.

Reply:
thank you that is extremely helpful

Reply:
Can you give me an idea of "brief description of what it means to have a SklearnModel"?

Reply:
You've opted to use object oriented programming. That means that I can instantiate a `SklearnModel` and get some object. What is that object? What does it mean?

If there's no good reason this should be an object, then it shouldn't be a class.

Reply:
It is not essential for the sklearn classifier. But it does make it simpler to load the model in a way that doesn't store the trained classifier as variable and without calling paths mutiple times, as it an just store it as an attribute. Same goes for the seeds. For the hognet classifier, there are too many arguments to do it all at once, and since passing the outputs of other functions into other functions doesn't fix that much, then that leaves us with classes.

Reply:
Then the object stores a classifier and its parameters. Sound OK?

Reply:
I dont't really possess the vocabulary to defend the design decision, but it does make a lot more sense to me.

Reply:
To be clear, I think this design decision is fine.